ballerina/types.bal

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2025 WSO2 LLC (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/ai;
+import ballerina/sql;
+import ballerinax/postgresql;
+
+# Represents vector data without ID
+#
+# + id - Unique identifier
+# + embedding - Vector embedding array
+# + metadata - Optional metadata
+# + similarity - Optional similarity score
+type SearchResult record {
+    string id;
+    string embedding;
+    string metadata?;
+    float similarity?;
+};
+
+# Configuration for the vector store
+#
+# + host - The host of the database

```

Comment:
Lets add comments for only the fields present, Otherwise there is a misleading API docs in the central

Reply:
Fixed in [382ef09](https://github.com/ballerina-platform/module-ballerinax-ai.pgvector/pull/1/commits/382ef091e41318084964e93daadc1a8e49ff31b2)